File: src/gtfs/models/StopTime.py
from pydantic import BaseModel, ConfigDict


class StopTime(BaseModel):
    """GTFS stop time."""
    model_config = ConfigDict(frozen=True)

    trip_id: str
    stop_id: str
    arrival_time: int  # seconds since midnight
    departure_time: int  # seconds since midnight
    stop_sequence: int

File: src/main/kotlin/com/raptor/gtfs/models/StopTime.kt
package com.raptor.gtfs.models

import kotlinx.serialization.Serializable

@Serializable
data class StopTime(
    val trip_id: String,
    val stop_id: String,
    val arrival_time: Int,
    val departure_time: Int,
    val stop_sequence: Int
)
